help/destinations/catalog/data-partners/merkury-enterprise-connections.md

```diff
@@ -0,0 +1,66 @@
+---
+keywords: Experience Platform;home;popular topics;destinations;connectors;destination connectors;destinations sdk;sdk;SDK
+title: Merkury Enterprise Connections Documentation self-service template for the UI
+description: Learn how to create a Merkury Enterprise Identity Resolution source connection using the Adobe Experience Platform UI.
+hide: true
```

Comment:
are you planning to hide these docs from readers and from search engines? wondering why you used the hide and hidefromtoc metadata on lines 5 and 6.